README.md

```diff
@@ -41,6 +41,11 @@ or open your `config/app.php` and add this line in `providers` section
 L5Swagger\L5SwaggerServiceProvider::class,
 ```
 
+## Swagger annotations and generating documentation
+In order to generate the Swagger documentation for your API, Swagger offers a set of annotations to declare and manipulate the output. These annotations can be added in your controller. An example can [be found here](https://github.com/DarkaOnLine/L5-Swagger/blob/master/tests/storage/annotations/Swagger/Anotations.php). 
```

Comment:
Annotations can be left not only on controllers. It can be also added in:
- separate file
- models
- etc.

Maybe link to the swagger-php pet-store example? `https://github.com/zircote/swagger-php/tree/3.x/Examples`

Maybe a link to official Sweagget/OpenApi documentation?

Reply:
See new commit.